### DISCUSSION TITLE
How to disable Database-specific named prefix auto-completion?

### DISCUSSION THREAD

**Discussion by sk33wiff** [dbeaver/dbeaver#9340] [Q&A] [answered]
## System information: 
- Ubuntu 20.04
- DBeaver Version 7.1.3.202007192031

#### Connection specification:
- mysql:mysql-connector-java:5.1.44

#### Describe your question:
I have multiple MySQL DB connections set.
Curiously, for just one of them, the db name always gets included as a prefix when using the auto complete feature for table names.

e.g.
```sql
select * from mydb.mytable;
```

How can I turn that off? so whenever I'l writing SQL statements using the auto complete feature it would result instead in following: 

```sql
select * from mytable;
```

**Comment by kseniiaguzeeva** [comment, marked as answer]
You need to disable **Use long object names (always use schema/catalogue)**.
For this press Settings btn in SQL Editor window and go to Editors>SQL Editor>SQL Completion/Folding
![dbeaver_hkMKEka9HU](https://user-images.githubusercontent.com/51405061/88153339-f2e5a980-cc0d-11ea-881e-faacbf1afd07.png)
